Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/category/PodcastCategory.kt

```diff
@@ -139,7 +139,7 @@ fun EpisodeListItem(
                 top.linkTo(parent.top)
                 centerHorizontallyTo(parent)
 
-                width = Dimension.fillToConstraints
+                width = androidx.compose.foundation.layout.Dimension.fillToConstraints
```

Comment:
import

Reply:
Nope, using this means we can suppress only the function, not the entire file.